common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -400,6 +400,16 @@ private ErrorStrings() {
             "This device configuration was changed on the server. " +
                     "Please return the device to your administrator to restart the registration of this device.";
 
+    /**
+     * The device is registered with precreate/userless WPJ, and its registration was deleted by the admin.
+     * This is an irrecoverable error, and the admin has to re-prep the device.
+     * */
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_CODE =
+            "userless_device_deleted_on_server";
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE =
+            "This device configuration was changed on the server. " +
```

Comment:
Is configuration change same as device deleted?

Reply:
I guess I could just say that it's deleted.